Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Built from Node latest Alpine
+FROM node:10.0
```

Comment:
This isn't pulling alpine version, rather you would use `node:10.0-alpine`

<img width="1081" alt="screen shot 2019-02-13 at 10 58 06 am" src="https://user-images.githubusercontent.com/1680159/52744811-bbd38900-2f81-11e9-937d-6f417c7ddea5.png">

You can see the difference in sizes.